docker/add_gitbase.py

```diff
@@ -17,5 +18,32 @@ def get_or_create_gitbase_db():
     db.session.add(dbobj)
     db.session.commit()
 
+    return dbobj
+
+
+def add_gitbase_tables():
+    dbobj = get_or_create_gitbase_db()
+    TBL = ConnectorRegistry.sources['table']
+    for table in dbobj.all_table_names_in_schema('gitbase'):
+        # table_name should match the one in the datasource for fetch_metadata to work
+        if db.session.query(TBL).filter_by(table_name=table).first():
+            continue
+        if db.session.query(TBL).filter_by(table_name='gitbase.' + table).first():
+            continue
+
```

Comment:
I don't understand, so we have to tables for each `gitbase` table? One with `gitbase.` prefix and one without?

Reply:
No, we have only one with prefix.
But to create it we first need to create it without prefix and then rename.

Explanation why: https://src-d.slack.com/archives/C7TB5NEDN/p1556710827229300

Reply:
Thanks!